lib/src/icons/validationcheck.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-icon--validation-check {
+  vertical-align: -0.125em;
```

Comment:
I'm wondering if we should just add this to `.nx-icon`

Reply:
> I'm wondering if we should just add this to `.nx-icon`

It seems to pop up a lot so I moved it here: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/d0e175eb0abc97c26d1ce9f69d6d05aeecd1a92e